### ISSUE TITLE
[4.0] Modal grey background not full width

### ISSUE THREAD

**Issue by PhilETaylor** [joomla/joomla-cms#19224]
### Steps to reproduce the issue

4.0-dev install
login frontend as super admin
click Template Settings
Click Select on the logo media selector 

### Expected result

Full width grey background under modal

### Actual result

grey background clipped at edges

(Note Im on HALF a 27" screen)

<img width="1596" alt="screen shot 2017-12-29 at 18 14 17" src="https://user-images.githubusercontent.com/400092/34444197-449d4f02-ecc4-11e7-8464-a5ccfb8f30b2.png">

**Comment by franz-wohlkoenig** [comment]
@PhilETaylor got same Screen but don't understand the Issue.

**Comment by ghost** [comment]
Issue confirmed:

![bildschirmfoto 2017-12-29 um 19 30 31](https://user-images.githubusercontent.com/8235763/34444504-e93f125c-ecce-11e7-8fe4-4ca4a803e777.jpg)

### System information
4.0.0-dev
Sample Data
macOS Sierra, 10.12.6
Firefox 57 (64-bit)
#### MAMP 4.2
- Apache-Server
- PHP 7.0.22
- MySQLi 5.6.35

**Comment by C-Lodder** [comment]
@ciar4n this is causing the issue: https://github.com/joomla/joomla-cms/blob/4.0-dev/templates/cassiopeia/scss/blocks/_layout.scss#L11

As far as I can tell, all child element of the `<body>` use the `.container-*` class, so seeing as you added:

https://github.com/joomla/joomla-cms/blob/4.0-dev/templates/cassiopeia/scss/blocks/_css-grid.scss#L22

can we remove `max-width: 1080px + ($cassiopeia-grid-gutter * 4);`?

**Comment by anuragteapot** [comment]
@C-Lodder if we remove <code>max-width: 1080px + ($cassiopeia-grid-gutter * 4);</code> is this issue solve ??<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/19224">issues.joomla.org/tracker/joomla-cms/19224</a>.</sub>

**Comment by C-Lodder** [comment]
@Anu1601CS - yes this does resolve the issue, however Ciaran wrote the css-grid, so he may have added this for a reason. Don't want to remove it until I get confirmation from him. If it needs to stay, then we can replace this:

```
.site {
  background-color: #fafafa;

  > div {... }
}
``` 

with:
```
.site {
  background-color: #fafafa;

  > div:not(.modal-backdrop) {... }
}
``` 

although I'm not a fan of that approach

**Comment by anuragteapot** [comment]
@C-Lodder ok thanks
What about at this line
https://github.com/joomla/joomla-cms/blob/125821fcc9bd5c2f44ed7332484a7645844ded1f/templates/cassiopeia/css/template.css#L8814<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/19224">issues.joomla.org/tracker/joomla-cms/19224</a>.</sub>

**Comment by C-Lodder** [comment]
That's what the SCSS compiles to. We don't edit CSS files, we edit the SCSS files then compile it

**Comment by anuragteapot** [comment]
ok thanks<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/19224">issues.joomla.org/tracker/joomla-cms/19224</a>.</sub>

**Comment by C-Lodder** [comment]
PR: https://github.com/joomla/joomla-cms/pull/19241

**Comment by ghost** [comment]
closed as having Pull Request #19241
